### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by Jexordexan** [Jexordexan/vue-slicksort#139]
We should implement an intuitive keyboard control for the sorting.

The first list item should be the only focusable element.
By pressing `up`, `down`, `left`, `right`, you control the current selection.
By pressing `enter` or `space` you "grab" the current item.
By pressing `up`, `down`, `left`, `right`, you "move" the item.
By pressing `enter` you "drop" the item, or `esc` to cancel.
By pressing `tab` you exit the container.

https://medium.com/salesforce-ux/4-major-patterns-for-accessible-drag-and-drop-1d43f64ebf09
